src/qt/transactionview.cpp

```diff
@@ -592,6 +592,22 @@ void TransactionView::focusTransaction(const QModelIndex &idx)
     transactionView->setFocus();
 }
 
+void TransactionView::focusTransaction(const QString &hash)
+{
+    if(!transactionProxyModel)
+        return;
+
+    const QModelIndexList results = this->model->getTransactionTableModel()->match(
```

Comment:
IIRC this performs linear search. Probably something worth to optimise in case of bigger wallets.

Reply:
Are transactions in reverse chronological order? In that case it should be pretty quick, unless some race condition causes the new transaction to be missing.

Reply:
Transactions are *displayed* in whatever order the user chooses. I'm not sure what order is internally used, however...

Reply:
I created a fresh wallet and then called `generate 10000`. I didn't notice any delay when sending transactions, although my machine is relatively fast. I also tried reversing transaction order in the view.